ros/src/computing/planning/decision/packages/decision_maker/CMakeLists.txt

```diff
@@ -80,11 +79,21 @@ install(TARGETS decision_maker_node
         RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
         )
 
-install(DIRECTORY launch
-        DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}/launch/
+## Install project namespaced headers
+install(DIRECTORY include/
```

Comment:
@s-azumi the install directive for the _include_ folder will generate _include/decision_maker_ in the _install_ folder. Can you create a _decision_maker_ folder inside _include_ and add the headers? The directive would then be:
```
install(DIRECTORY include/${PROJECT_NAME}/
        DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
        PATTERN ".svn" EXCLUDE
)
```

Reply:
Thanks for your review, I fixed it.